Sprint 1 Task 2 - Kudos feature

DamianWieszczak opened the pull request.

piowit commented on app/helpers/kudos_helper.rb:
```diff
@@ -0,0 +1,2 @@
+module KudosHelper
```
Starajmy nie dodawać do PR pustych plików które nic nie robią, a są genreowane przez generatory

piowit commented on app/models/employee.rb:
```diff
@@ -3,4 +3,7 @@ class Employee < ApplicationRecord
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
+
+  has_many :received_kudos, class_name: 'Kudo', foreign_key: 'receiver_id', dependent: :destroy, inverse_of: :employee
+  has_many :given_kudos, class_name: 'Kudo', foreign_key: 'giver_id', dependent: :destroy, inverse_of: :employee
```
hej `inverse_of` podpowiada railsom co jest odwrotnością relacji
przykład: `Kudo has many given_kudos` -> odwrotnością tego jest `Kudo belongs to giver`
a więc w tym przypadku powinieneś wstawić tutaj `:giver` a nie employee, oraz analogicznie dla receivera

piowit commented on app/views/home/index.html.erb:
```diff
@@ -1,2 +1,14 @@
+  <div class="card-footer">
+    <%= Time.now.strftime("%d of %B, %Y") %>
+  </div>
+</div>
```
staraj się dodawać pustą linijkę na końcu plików, można ustawić sobie IDE aby robiło to za ciebie (szukaj pod add new line)

piowit commented on app/views/kudos/index.html.erb:
```diff
@@ -0,0 +1,20 @@
+        <h6 class="card-title">
+          Content: <%= kudo.content %>
+        </h6>
+          Receiver: <p class="card-text"><%= kudo.receiver.email %></p>
+          Giver: <p class="card-text"><%= kudo.receiver.email %></p>
```
tutaj jest błąd w 12 linii

piowit reviewed (commented):
Looks good :fire:
I suggested some changes

DamianWieszczak commented:
Making corrections to task 2 according to the mentor's recommendation:
- removed empty helpers from generators
- added option to automatically add a blank line at the end
- fixed duplicate display of kudo.receiver.email on kudo.giver.email
- the naming in the employee.rb file related to inverse_of has been corrected, incorrectly assumed inverse_of: :employee has been changed to inverse_of: :receiver and inverse_of: :giver

piowit reviewed (approved):
looks good to me :)
